thirdparty/workspace.bzl

```diff
@@ -1,4 +1,16 @@
-def maven_dependencies(callback):
+def declare_maven(hash):
```

Comment:
can you delete load.bzl and the reference to it in WORKSPACE, now that it's not needed anymore?

Reply:
👍 Updated the PR.